python/pyarrow/parquet.py

```diff
@@ -591,23 +592,32 @@ class ParquetManifest(object):
 
     """
     def __init__(self, dirpath, filesystem=None, pathsep='/',
-                 partition_scheme='hive'):
+                 partition_scheme='hive', thread_pool=None):
```

Comment:
@pitrou Wouldn't this be a use case for the ThreadPool you have introduced recently?

Reply:
If this were C++, then yes ;-)